package.json

```diff
@@ -54,6 +54,7 @@
     "eslint-plugin-jsx-a11y": "^6.6.0",
     "eslint-plugin-react": "^7.30.1",
```

Comment:
- [ ] Please there is an issue in the code 
![image](https://user-images.githubusercontent.com/74744787/179090787-b7030c34-7119-4c4c-881b-bee1cec51379.png)